vendor/shims/countdown.js

```diff
@@ -0,0 +1,9 @@
+(function() {
+  function vendorModule() {
+    'use strict';
+
+    return { 'default': self['countdown'] };
+  }
+
+  define('countdownjs', [], vendorModule);
+})();
```

Comment:
do you mind explaning this too? (probably just the way to get something from npm/vendor instead of bower)

Reply:
This just an ES6 shim to make `countdown.js` available via `import`. Basically the same as the `vendor/ember-countdown.js` file that was there, but generated using ember-cli using `ember generate vendor-shim`.